lib/assets/SnapshotHelper.swift

```diff
@@ -66,11 +66,26 @@ class Snapshot: NSObject {
         if waitForLoadingIndicator {
             waitForLoadingIndicatorToDisappear()
         }
-
-        print("snapshot: \(name)") // more information about this, check out https://github.com/fastlane/snapshot
-
+        
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)